aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -197,6 +197,11 @@ module aptos_framework::stake {
         validators: vector<IndividualValidatorPerformance>,
     }
 
+    // The beneficiary address for the operator.
+    struct Beneficiary has key {
```

Comment:
As discussed, we should move this to staking_contract and delegation_pool. It's a bit redundant but keep the abstraction clear as stake.move is not aware of commission.

Reply:
Done moving this resource to `staking_contract` and `delegation_pool` separately.